src/test/java/uk/gov/hmcts/befta/data/CollectionVerificationConfigTest.java

```diff
@@ -40,4 +59,170 @@ void testToString() {
 		assertNotNull(other.toString());
 	}
 
+	/**
+	 * Test method for {@link uk.gov.hmcts.befta.data.CollectionVerificationConfig#getVerificationConfigFrom(Collection)} ()}.
+	 */
+	@Test
+	void testVerificationConfigFromNullCollection() {
+		assertEquals(CollectionVerificationConfig.DEFAULT, CollectionVerificationConfig.getVerificationConfigFrom(null));
+	}
+
+	/**
+	 * Test method for {@link uk.gov.hmcts.befta.data.CollectionVerificationConfig#getVerificationConfigFrom(Collection)} ()}.
+	 */
+	@Test
+	void testVerificationConfigFromEmptyCollection() {
+		assertEquals(CollectionVerificationConfig.DEFAULT,
+				CollectionVerificationConfig.getVerificationConfigFrom(unmodifiableCollection(Collections.emptyList())));
+	}
+
+	/**
+	 * Test method for {@link uk.gov.hmcts.befta.data.CollectionVerificationConfig#getVerificationConfigFrom(Collection)} ()}.
+	 */
+	@Test
+	void testVerificationConfigReturnsDefaultFromListCollectionWhereirstElementIsCollectionVerificationConfig() {
```

Comment:
typo in method name.

Reply:
Cheers - they are way too long...Need to sort

Reply:
resolved in https://github.com/hmcts/befta-fw/pull/137/commits/25a5aaeaaf5b1f5144499ab6112c0f2ee82e8329